### ISSUE TITLE
fail import utils in cab/montage

### ISSUE THREAD

**Issue by Fil8** [SpheMakh/Stimela#300]
I upgraded `Stimela` in the new version of `meerkathi`. 

I get an error when I try to run the `cab/montage`. The issue is in importing the library `utils`.

```
running: docker create --user 1001:1001  -v /home/maccagni/programs/fornaxa-venv/local/lib/python2.7/site-packages/stimela:/scratch/stimela:ro -v /home/maccagni/Projects/MFS/FornaxA/MeerKAT/fa3/stimela_parameter_files:/configs:ro -v /home/maccagni/Projects/MFS/FornaxA/MeerKAT/fa3/msdir:/home/maccagni/msdir:rw -v /home/maccagni/Projects/MFS/FornaxA/MeerKAT/fa3/input:/input:ro -v /home/maccagni/Projects/MFS/FornaxA/MeerKAT/fa3/output:/home/maccagni/output:rw  -e CONFIG=/configs/2-139872121013568154019832236.json -e MSDIR=/home/maccagni/msdir -e INPUT=/input -e HOME=/home/maccagni/output -e OUTPUT=/home/maccagni/output -e LOGFILE=/home/maccagni/output/log-2.txt  --name 2-139872121013568154019832236 --shm-size 1gb maccagni_cab/montage 
dbfd512682e49eabc3bf5ae2c89fa635bccbac6bf80d1babeea56a580262b1cc
running: docker start -a 2-139872121013568154019832236
Traceback (most recent call last):
  File "/code/run.py", line 5, in <module>
    import utils
ImportError: No module named utils
2018-10-22 10:52:03,693 - STIMELA-11 - INFO - Recipe execution failed while running job 2
INFO:STIMELA-11:Recipe execution failed while running job 2
2018-10-22 10:52:03,693 - STIMELA-11 - INFO - Completed jobs : ['query_source_catalog', '1']
INFO:STIMELA-11:Completed jobs : ['query_source_catalog', '1']
```

I think this is due to these lines in `run.py`:

```
sys.path.append('/utils')
import utils
```

what should I do?

**Comment by SpheMakh** [comment]
Fixed in dbda4d1 (see branch `scratch`)
 Please test and let me know if it works

**Comment by SpheMakh** [comment]
Fixed
